test/extended/networking.sh

```diff
@@ -318,6 +318,13 @@ else
 
   os::util::environment::setup_tmpdir_vars "test-extended/networking"
 
+  # Allow setting $JUNIT_REPORT to toggle output behavior
+  if [[ -n "${JUNIT_REPORT:-}" ]]; then
+    export JUNIT_REPORT_OUTPUT="${LOG_DIR}/raw_test_output.log"
```

Comment:
This is for the logging from the `os::cmd` tests, if there are any, not for the jUnit from Ginkgo.